test/async-hooks/test-async-local-storage-thenable.js

```diff
@@ -1,3 +1,4 @@
+// Flags: --expose-internals
```

Comment:
Why is this needed now?

Reply:
It's not. Deleted it now. Just some leftovers from iteration. 😅